### DISCUSSION TITLE
Uploading a file

### DISCUSSION THREAD

**Discussion by LegoMashine** [community/community#140544] [GitHub Education]
### Hi everyone,

Hello everyone, I tried uploading my files in order to apply for GitHub Education, but am unable to do so since both my pc and my phone require me to take a picture from a webcam/camera. Does anyone know why this is happening and how to fix it?

**Comment by SaswataGA** [comment]
Maybe these options can be helpful:
1. Try to Change the browser or open the incognito mode to check whether the upload option is available.
2. Investigate the browser settings to ensure that it is not trying to open the camera.
3. If it still does not work, contact GitHub support so that they assist you in uploading your files.

That should do the trick! I hope so!
